src/cassandra/row.rs

```diff
@@ -21,27 +22,28 @@ use std::fmt::Display;
 use std::fmt::Formatter;
 use std::iter;
 use std::iter::IntoIterator;
+use std::marker::PhantomData;
 use std::os::raw::c_char;
 
 /// A collection of column values. Read-only, so thread-safe.
-pub struct Row(*const _Row);
+pub struct Row<'a>(*const _Row, PhantomData<&'a CassResult>);
```

Comment:
:+1: Clearly the lifetime should be bound to the result.

Reply:
Yes :-)